testing/py3-us/APKBUILD

```diff
@@ -11,18 +11,15 @@ license="BSD-3-Clause"
 replaces="py-us" # for backwards compatibility
 provides="py-us=$pkgver-r$pkgrel" # for backwards compatibility
 depends="py3-jellyfish py3-setuptools python3"
```

Comment:
Looks incorrect to me. The makedepends should remain, the overlap should be removed from depends.

Would also be great if check was added - https://github.com/unitedstates/python-us/tree/master/tests

Reply:
No, assuming dependency distribution was correct before then you should ever remove from makedepends when against depends.

Reply:
Forgot to paste link for reqs - https://github.com/unitedstates/python-us/blob/master/requirements.txt